autoshop/core/forms.py

```diff
@@ -26,11 +26,11 @@ class Meta:
         model = User
         fields = ['first_name', 'last_name', 'username', 'email', 'password']
         help_texts = {
-            'first_name': 'Enter first name:',
-            'last_name': 'Enter last name:',
-            'username': 'Enter username:',
-            'email': 'Enter email:',
-            'password': 'Enter password:',
+            'first_name': 'Enter first name',
+            'last_name': 'Enter last name',
+            'username': 'Enter username',
```

Comment:
We need to also tell users that it cannot be change later.
Check `edX` register for example: https://courses.edx.org/register